src/js/messaging/config.js

```diff
@@ -17,6 +17,10 @@ module.exports = {
           'Content-Type': 'application/json'
         })
       },
+     // For sending binary data requests. Assumes use of FormData API.
+      postFormData: {
+        method: 'POST'
```

Comment:
This should have the header for camel casing too, so that the id from the JSON response can be used for the link displayed in the success alert. Maybe also rename the other setting to `postJson` for clear distinction.